com.trekglobal.idempiere.rest.api/src/com/trekglobal/idempiere/rest/api/model/MOIDCService.java

```diff
@@ -89,6 +90,9 @@ public class MOIDCService extends X_REST_OIDCService implements ImmutablePOSuppo
 	/** HTTP header for AD_Language */
 	public static final String LANGUAGE_HEADER = "X-ID-Language";
 	
+	/** HTTP header for AD_Org.Value. */ 
+	public static final String IDTOKEN_HEADER = "IdToken";
```

Comment:
Should use X- prefix for custom http header. My suggestion is to use X-ID-IdToken (X for custom and ID for iDempiere).

Reply:
Changed to X-ID-IdToken